gateway_code/open_nodes/node_pycom.py

```diff
@@ -37,13 +39,28 @@
 }
 PYCOM_FLASH_ERASE_SEQUENCE = (
     b"import os\r\n",
-    b"os.mkfs('/flash')\r\n",  # Erase the flash
+    b"os.fsformat('/flash')\r\n",  # Erase the flash
 )
 PYCOM_RESET_SEQUENCE = (
     b"import machine\r\n",
     b"machine.reset()\r\n",
 )
 
+PYCOM_UPDATE_BIN = "sudo /usr/bin/python3 " \
+    + "/usr/local/share/pycom/eps32/tools/fw_updater/updater.py"
```

Comment:
The sudo command is not available on gateway images. The gateway instance is running with the www-data user with limited rights. Why do you need `sudo` ? Do you think it could work without ?